internal/pubsub/driver/driver.go

```diff
@@ -61,10 +61,8 @@ type Topic interface {
 	// Only the Body and (optionally) Metadata fields of the Messages in ms
 	// should be set by the caller of SendBatch.
```

Comment:
"should be set" -> "will be set"? This documentation is really for the driver implementor, not the caller.

Reply:
+1

Reply:
Done.